batch/test/test_batch.py

```diff
@@ -936,6 +936,7 @@ def test_authorized_users_only():
         (session.post, '/api/v1alpha/billing_limits/foo/edit', 401),
         (session.get, '/api/v1alpha/batches/0/jobs/0', 401),
         (session.get, '/api/v1alpha/batches/0/jobs/0/log', 401),
+        (session.get, '/api/v1alpha/batches/0/jobs/0/resource_usage', 401),
```

Comment:
Sorry, I missed these messages! How do I do this, do I need to manually add things to the batch_client so it knows how to get the resource usage? Trying to follow an existing pattern, but not really finding anything

Reply:
No worries! I think you can look at `batch_client.Job.log` as a good example. It's just a couple shims around the API call and returns the output JSON. This is in the "low-level" client that just wraps the HTTP API so this doesn't involve and user-facing changes or documentation, etc. If you could do that and just add a test like this but instead of loading the log loads the dataframes that would be amazing:

```python
def test_job(client: BatchClient):
    b = create_batch(client)
    j = b.create_job(DOCKER_ROOT_IMAGE, ['echo', 'test'])
    b.submit()

    status = j.wait()
    assert 'attributes' not in status, str((status, b.debug_info()))
    assert status['state'] == 'Success', str((status, b.debug_info()))
    assert status['exit_code'] == 0, str((status, b.debug_info()))
    assert j._get_exit_code(status, 'main') == 0, str((status, b.debug_info()))
    job_log = j.log()
    assert job_log['main'] == 'test\n', str((job_log, b.debug_info()))
```